pkg/api/graph/graph.go

```diff
@@ -122,7 +122,7 @@ var DefaultNamer Namer = namer{}
 func (namer) ResourceName(obj interface{}) string {
 	switch t := obj.(type) {
 	case uniqueNamer:
-		return t.UniqueName()
+		return t.UniqueName().UniqueName()
```

Comment:
I added a `String()` method for the `UniqueString` type and removed the `UniqueName()` method (which was not being used anywhere other than the local instance I temporarily had) ... seemed a bit cleaner than typecasting to a string.